README.md

```diff
@@ -2,105 +2,99 @@
 
 <div align="center">
 <h1>Autometrics</h1>
-<a href="https://github.com/autometrics-dev/autometrics-ts/actions?query=branch%3Amain"><img src="https://github.com/autometrics-dev/autometrics-ts/actions/workflows/check_wrappers.yml/badge.svg?event=push&branch=main" alt="Autometrics CI status" /></a>
+<a href="https://github.com/autometrics-dev/autometrics-ts/actions?query=branch%3Amain"><img src="https://github.com/autometrics-dev/autometrics-ts/actions/workflows/ci.yml/badge.svg?event=push&branch=main" alt="Autometrics CI status" /></a>
 <a href="https://opensource.org/licenses/MIT" rel="nofollow"><img src="https://img.shields.io/npm/l/@autometrics/autometrics" alt="License"></a>
 <a href="https://discord.gg/MJr7pYzZQ4" rel="nofollow"><img src="https://img.shields.io/discord/950489382626951178?label=Discord&logo=discord&logoColor=white" alt="discord server"></a>
 </div>
 
 <hr />
 
-> A TypeScript port of the Rust
-> [autometrics-rs](https://github.com/fiberplane/autometrics-rs) library
+Autometrics library provides a wrapper for functions and decorator
+for classes and methods to instrument them with the most useful
+metrics: request rate, error rate, and latency. It then uses the instrumented
+function names to generate Prometheus queries so you don’t need to hand-write
+complicated PromQL.
 
-**A library that makes it easy to understand the error rate, response time, and
-production usage of any function in your code.** Jump straight from your IDE to
-live Prometheus charts for each HTTP/RPC handler, database method, or other
-piece of application logic.
+[Learn more about Autometrics here](https://autometrics.dev/).
 
-Autometrics for TypeScript provides a wrapper function and a decorator that can
-create Prometheus or OpenTelemetry metrics for your functions and class methods
-throughout your code base, as well as a language service plugin that will write
-corresponding Prometheus queries for you.
-
-See [Why Autometrics?](https://github.com/autometrics-dev#why-autometrics) for
-more details on the ideas behind autometrics
-
-![AutometricsTS demo](./assets/autometrics-ts-demo.gif)
 
 ## Features
 
-- ✨ `autometrics` wrapper instruments any function or class method to track its
+- ✨ `autometrics()` wrapper / `@Autometrics()` instruments any function or
+class method to track its
   most useful metrics
-- 🌳 Works in NodeJS and browser environments (Deno and serverless support coming soon)
+- 🌳 Works in NodeJS, experimental support for browser and Deno environments
 - 💡 Writes Prometheus queries so you can understand the data generated without
   knowing PromQL
 - 🔗 Injects links to live Prometheus charts directly into each function's doc
+- 🔍 Helps you to [identify commits](https://docs.autometrics.dev/typescript/adding-version-information) that introduced errors or increased latency
+- 🚨 Allows you to [define alerts](https://docs.autometrics.dev/typescript/adding-alerts-and-slos) using SLO best practices directly in your source code
```

Comment:
Similar to the Rust readme redo, I would separate the benefits from the advanced features (and name the first section Benefits)